near2eth/near2eth-block-relay/index.js

```diff
@@ -70,7 +71,7 @@ class Near2EthRelay {
         // The finalized block is not immediately available so we wait for it to become available.
         let lightClientBlock = null
         let currentValidators = null
-        while (!lightClientBlock || Object.keys(lightClientBlock).length === 0) {
+        while (_.isEmpty(lightClientBlock)) {
```

Comment:
I would prefer not to have this `_.METHOD_NAME` pattern in our code. Please change

Reply:
fixed at 77ef641809463892d74d6e494ffc0bf607298ed0